lambdas/reset-identity/src/main/java/uk/gov/di/ipv/core/resetidentity/ResetIdentityHandler.java

```diff
@@ -166,10 +186,10 @@ private void sendIpvVcResetAuditEvent(
 
     // Try to get the user's name from their VCs. It's not the end of the world if this fails so
     // just return null in that case.
-    private String getUserName(List<VcStoreItem> credentials) {
+    private String getUserName(List<VcStoreItem> credentials, boolean checkEvidence) {
```

Comment:
As before, please rename this to `getUnconfirmedUserName()` and remove the `checkEvidence` flag from here too. This method should always pass `false` to `findIdentityClaim()`